docs-dev/NODE_METRICS.md

```diff
@@ -0,0 +1,41 @@
+# Node Metrics
+
+By default, Ceramic Nodes publish some limited metrics about their state to a model on the Ceramic Network.
+
+## Metric Data Published
+
+The data published is defined in the [@ceramicnetwork/node-metrics](https://www.npmjs.com/package/@ceramicnetwork/node-metrics) package.
+
+Currently, this includes the following about the Ceramic node:
+
+  - DID used to log into the Ceramic Anchor Service
+  - IP address, if known
+  - Peer ID (currently on IPFS)
+  - Ceramic Version
+  - IPFS Version
+
+As well as the following dynamic metrics published periodically (by default once per minute):
+
+  - total Pinned streams
+  - total Indexed models
+  - current pending requests to CAS
+  - mean Anchor request age
+  - max Anchor request age
+  - number of created requests in the last window
```

Comment:
is this still about anchor requests or is this some other concept of request?  Should this say "number of writes" instead?

Reply:
this is about anchor requests.  that is literally what it is measuring but its fine if we say "writes" - up to you

Reply:
if it's measuring anchor requests i'd rather it say anchor requests 😅